add relhum_water outputs

cyschneck opened the pull request:
Add `relhum_water` output values for use in applications

Output built from NCL script: 
```
; relhum_water
; Adapted from https://www.ncl.ucar.edu/Document/Functions/Built-in/relhum_water.shtml

; ncl -n relhum_water.ncl >> relhum_water_output.txt

print("Temperature (K), Mixing Ratio (KG/KG), Pressure (Pa)")
do tk=273,373
	do mr=1,100
		do ppa=10000,15000,500
			begin
				rh = relhum_water(tk, mr, ppa)
				print (tk + "," + mr + "," + ppa + "," + rh)
			end
		end do
	end do
end do
```
Varied out the range of pressure to prevent jupyter from crashing due to size

cyschneck commented:
The accuracy between NCL and geocat-comp will vary since the vapor pressure constant of water at 0 degrees C in Pa is different between the packages [(Example in WIP receipt)](https://ncar.github.io/geocat-applications/_preview/196/ncl/receipts/meteorology.html#id9)
- NCL: es0 = 6.11
- geocat-comp: es0 = 6.1128

kafitzgerald reviewed (commented):
I think I might've missed this in reviewing prior examples, but the range of values for mixing ratios seems rather high.  A reasonable range probably looks more like 0 - 0.05 kg/kg.  Maybe these values were g/kg?

Otherwise, looks good.

cyschneck commented:
> I think I might've missed this in reviewing prior examples, but the range of values for mixing ratios seems rather high. A reasonable range probably looks more like 0 - 0.05 kg/kg. Maybe these values were g/kg?
> 
> Otherwise, looks good.

I believe you are right, I updated it and added the changes here. New script for future reference:
```
; relhum_water
; Adapted from https://www.ncl.ucar.edu/Document/Functions/Built-in/relhum_water.shtml

; ncl -n relhum_water.ncl >> relhum_water_output.txt

print("Temperature (K), Mixing Ratio (KG/KG), Pressure (Pa)")
do tk=273,373
	do mr=0.01,0.05,0.01
		do ppa=10000,15000,500
			begin
				rh = relhum_water(tk, mr, ppa)
				print (tk + "," + mr + "," + ppa + "," + rh)
			end
		end do
	end do
end do
```
